Dockerfile.dist

```diff
@@ -0,0 +1,42 @@
+# This dockerfile builds a base image that can run a binary version of S2E.
+# First, build S2E using the Dockerfile, then copy /opt/s2e from the resulting image
+# to this image.
+
+FROM ubuntu:16.04
+MAINTAINER Volodymyr Kuznetsov <vova@cyberhaven.io>
+
+# i386 packages are required by IDAPro
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ precise-pgdg main 9.4" \
```

Comment:
caching makes sense for very long operations. I don't think adding one line to a file warrants it. It would actually create more overhead.